### DISCUSSION TITLE
Highlight usages of local values when selected

### DISCUSSION THREAD

**Discussion by StachuDotNet** [darklang/dark#4614] [Suggestions]
If my text cursor is within some Dark variable name, I'd like that variable name to be 'highlighted' everywhere else in the surrounding of code.

Most immediately, this came up when tidying some code - without such a highlight, it's hard to immediately tell if I'm able to remove the line of code assigning the value. (are there any usages?)

**Comment by pbiggar** [comment]
I think this used to work, implying there's code in there to support this but that it broke at some point.
